lib/puppet/provider/gnupg_key/gnupg.rb

```diff
@@ -32,12 +25,7 @@ def remove_key
     elsif resource[:key_type] == :both
       command = "gpg --batch --yes --delete-secret-and-public-key #{fingerprint}"
     end
```

Comment:
http://stackoverflow.com/questions/948135/how-can-i-write-a-switch-statement-in-ruby